api/golang/engine/lib/enclave/consts.go

```diff
@@ -1,3 +1,3 @@
 package enclave
 
-const AllowedEnclaveNameCharsRegexStr = `^[-A-Za-z0-9._]{1,63}$`
+const AllowedEnclaveNameCharsRegexStr = `^[-A-Za-z0-9._]{1,59}$`
```

Comment:
isn't this 59 chars (not 60)? either is fine with me, but we should be clear

Reply:
maybe should also add a comment explaining _why_ we're making this particular length restriction

Reply:
lol facepalm tedi